source/spark_sql_migrations/README.md

```diff
@@ -0,0 +1,67 @@
+# Spark SQL Migrations
+
+## Usage
+
+```python
+
+import spark_sql_migrations.migrations.schema_migration_pipeline as schema_migration_pipeline
+from spark_sql_migrations.container import create_and_configure_container
+from spark_sql_migrations.models.spark_sql_migrations_configuration import (
+    SparkSqlMigrationsConfiguration,
+)
+from spark_sql_migrations.models.table import Table
+from spark_sql_migrations.models.schema import Schema
```

Comment:
Would it make sense to make all these parts that constitute the public API of the package available directly as `from spark_sql_migrations import a, b, c`? It would make it much easier to get an overview of the package as a consumer and find the relevant symbols. What are your thoughts?

You wouldn't need to change the project structure, but can simply add the relevant symbols to the root init file.

Reply:
Very interesting. I will try it out and change if it works :-)